test/modules/EdgeHubRestartTester/config/settings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "IOT_HUB_CONNECTION_STRING": {
+    "value": ""
+  },
+  "directMethodEnable": {
+    "value": "true"
+  },
+  "directMethodTargetModuleId": {
+    "value": "directMethodReceiver1"
+  },
+  "messageEnable": {
+    "value": "true"
+  },
+  "reportingEndpointUrl": {
+    "value": "http://localhost:5001"
+  },
+  "restartIntervalInMins": {
+    "value": "5"
+  },
+  "sdkRetryTimeoutMilisec": {
+    "value": "00:00:00.020"
+  },
+  "testDuration": {
+    "value": ""
```

Comment:
nit:provide default value; also for testStartDelay.

Reply:
what do you think `testDuration` should be?